scripts/local_testnet/README.md

```diff
@@ -15,45 +15,27 @@ make install-lcli
 
 ## Starting the testnet
 
-Start a local eth1 ganache server
-```bash
-./ganache_test_node.sh
-```
-
-Assuming you are happy with the configuration in `vars.env`, deploy the deposit contract, make deposits,
-create the testnet directory, genesis state and validator keys with:
+Modify `vars.env` as desired.
 
-```bash
-./setup.sh
-```
+Start a local eth1 ganache server plus boot node along with NODE_COUNT
+number of beacon nodes and VC_COUNT validator clients.
 
-Generate bootnode enr and start a discv5 bootnode so that multiple beacon nodes can find each other
-```bash
-./bootnode.sh
-```
+This script takes two options `-c VC_COUNT` and `-d DEBUG_LEVEL`.
+The options maybe in any order or absent in which case they take the default value specified.
+- VC_COUNT: the number of validator clients to create, default: `1`
```

Comment:
I think the default here should be `VC_COUNT == NODE_COUNT` as we are distributing the validators among all nodes equally. 
It might be easier to have this param in `vars.env`.
Also, having just 1/4th of the validators active would prevent finality, so we wouldn't want that to be default.